source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,95 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can select and convert one or more queries, stored procedures, 
+triggers, and views in bulk by enabling the multi-select toggle. 
+Bulk conversion creates a job that queues and executes each query 
+conversion from a single request.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
+
+- .. include:: /includes/fact-view-query-conversion-bulk-status.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      From the :guilabel:`Code Generation` tab, click the 
+      :guilabel:`Query Converter` pane.
+
+   .. step:: Enable multi-selection
+
+      Click the :guilabel:`Enable multi-selection` toggle to show the query 
+      selection pane.
```

Comment:
(Suggestion)
Possibly explain why you want to enable multi-selection? Perhaps this is self evident, but consider explaining the benefits here?

Reply:
@sarah-olson-mongodb My only hesitation here is that we define that benefit in the intro paragraph (I swapped the order of the sentences since last draft to start with the general feature and then describe the interaction mechanism):

> Bulk conversion creates a job that queues and executes each query 
> conversion from a single request. You can select and convert one 
> or more queries, stored procedures, triggers, and views in bulk 
> by enabling the multi-select toggle. 

Happy to clarify in the step as well if you think it makes sense.

Reply:
This explains how it works but not why you would do it, although arguably the "why" is pretty obvious. If we wanted to say something extra, it could be something like:

> Bulk query conversion allows you to convert a large number of queries from a single UI action, avoiding the need to convert each query individually.

Reply:
That makes sense. I took the copy provided and moved it to the intro paragraph, reworking the follow-up content a bit and adding a little more high-level detail to the section.